npm/react18/src/index.ts

```diff
@@ -23,20 +32,12 @@ export function mount (jsx: React.ReactNode, options: MountOptions = {}, rerende
       return root.render(reactComponent)
     },
     unmount,
+    cleanup,
   }
 
   return makeMountFn('mount', jsx, { ReactDom: ReactDOM, ...options }, rerenderKey, internalOptions)
 }
 
 export function unmount (options: UnmountArgs = { log: true }) {
-  const internalOptions: InternalUnmountOptionsReact18 = {
-    // type is ReturnType<typeof ReactDOM.createRoot>
-    unmount: (): boolean => {
-      root.unmount()
-
-      return true
-    },
-  }
-
-  return makeUnmountFn(options, internalOptions)
```

Comment:
The file below this (`_snapshots__`) has a surprisingly huge diff with a lot of red. Is this right? It seems to be missing the "summary" part of the CLI output.